accesshandler/openapi.yml

```diff
@@ -151,6 +151,27 @@ paths:
         name: providerId
         in: path
         required: true
+  "/api/v1/providers/{providerid}/validate":
```

Comment:
Why not just make this /api/v1/grants/validate POST?

The Grant request body contains all the information required for the Grant/validation